exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/package-info.java

```diff
@@ -32,6 +32,22 @@
  * batch" or "record batch." (But, in Drill, the term "record batch" also
  * usually means an operator on that set of records. Here, a row set is
  * just the rows &nash; separate from operations on that data.</dd>
+ * <dt>SingleRowSet (abstract)</dt>
```

Comment:
See general comment. Please do not combine the RowSet and ResultSetLoader classes into a single package: they present very different models used for different purposes.

Yes, it is confusing because there is no ResultSetReader at present, but there should be to handle schema changes, etc. across batches. RowSet handles only a single row set such as in tests. It was deliberately designed to NOT be ResultSetReader solution.

Reply:
Thanks!